src/components/Checkout/LoanPrice.vue

```diff
@@ -264,7 +264,6 @@ export default {
 
 	@include breakpoint(medium) {
 		height: inherit;
-		line-height: $medium-text-line-height;
```

Comment:
This change was required after updating foundation-sites and I believe it was due to some changes in one of their scss files. I've reviewed all the major Ui pages and all the stories in storybook and didn't find any other anomalies.